src/preferences/dialog/dlgprefreplaygaindlg.ui

```diff
@@ -246,6 +221,43 @@
     </widget>
    </item>
    <item>
+    <widget class="QGroupBox" name="groupBoxAnalysis">
+     <property name="title">
+      <string>Analysis</string>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout_3">
+      <item>
+       <widget class="QRadioButton" name="radioButtonRG2">
+        <property name="text">
+         <string>ReplayGain 2.0 (EBU R 128)</string>
```

Comment:
Is this sending mixed messages? (The only difference between RG2 and EBU R 128 is -18 LUFS reference vs -23, right?)